src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -31,6 +36,16 @@ trait JsonRpcHttpServer extends Json4sSupport {
 
   def corsAllowedOrigins: HttpOriginMatcher
 
+  val minRequestInterval: FiniteDuration
+
+  val latestTimestampCacheSize: Int
+
+  val latestRequestTimestamps: LruMap[RemoteAddress, Long]
```

Comment:
Maybe this is included in the above comment by Maxi, but also this can be derived from the config here without having to override it on it's sons